.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+  "recommendations": [
+    "arcanis.vscode-zipfs",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode"
```

Comment:
This shouldn't be needed right? Running eslint invokes prettier plus more.
```suggestion
```

Reply:
Correct